airbyte-ci/connectors/connectors_qa/README.md

```diff
@@ -0,0 +1,87 @@
+# Connectors QA
```

Comment:
Two questions here:

1. Does `connectors_qa` have to live in this particular directory, or is it just logical spot for it in the monorepo? I.e. could we just put it in the root of the repository instead?
2. If it's fully independent of the rest of connector_ops stuff (don't think so), is there a good overall readme and list of directories and what project is where? We have many things — registry service, airbyte-ci, etc.

Reply:
@natikgadzhi I put it there because it's a package used in the CI context. But we could put it at the root of the repo, I don't mind 😄 . 
It's dependent on a couple of other internal packages. I will explicitly list the dependency and why they exists in this README.